inc/plugins/MybbStuff/MyAlerts/templates/alert_row_popup.html

```diff
@@ -13,7 +13,7 @@
         </a>
     </td>
     <td class="{$altbg} alert__time">
-        {$alert['received_at']}<br />
+        <span class="smalltext">{$alert['received_at']}</span><br />
```

Comment:
As above: this styling does not seem to be relevant to this PR. Same question as above.